test/python/algorithms/evolvers/variational/calculators/expected_results/test_metric_calculator_expected1.py

```diff
@@ -0,0 +1,360 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Stores expected results that are lengthy."""
+from numpy import array
+
+correct_values_1 = [
+    array(
+        [
+            [
+                1.00000000e00 + 0.0j,
```

Comment:
Are these values copy+paste from a numerical run? If yes I think it would be better to have some analytic values to compare against (which I think we do in the QFI or QNG tests) 😛

Reply:
Would you help with providing them?

Reply:
I think there are some in the QFI/QNG tests we can use, @Zoufalc did some calculations if I remember correctly 😄